packages/common/src/api/tan-query/useAssociatedWallets.ts

```diff
@@ -0,0 +1,36 @@
+import { Id } from '@audius/sdk'
+import { useQuery } from '@tanstack/react-query'
+
+import { useAudiusQueryContext } from '~/audius-query/AudiusQueryContext'
+import { Chain } from '~/models'
+
+import { QUERY_KEYS } from './queryKeys'
+import { QueryOptions } from './types'
+import { useCurrentUserId } from './useCurrentUserId'
+
+export const useConnectedWallets = (options?: QueryOptions) => {
+  const { audiusSdk } = useAudiusQueryContext()
+  const { data: currentUserId } = useCurrentUserId()
+
+  return useQuery({
+    queryKey: [QUERY_KEYS.connectedWallets],
```

Comment:
we've been using helper fns (e.g. `getConnectedWalletsKey`) to make the keys. This keeps us from messing up the structure across references

also you probably want to include currentUserId in the key so when you switch accounts the wallets update?